CMakeLists.txt

```diff
@@ -45,14 +45,27 @@ set(BUILD_SHARED_LIBS OFF)
 # CRT Libraries
 add_subdirectory(crt/aws-c-common)
 if (UNIX AND NOT APPLE)
-    if (NOT (CMAKE_SYSTEM_PROCESSOR STREQUAL aarch64) AND
-        NOT (CMAKE_SYSTEM_PROCESSOR STREQUAL armv7l))
-        set(DISABLE_PERL ON)
+    if (NOT USE_OPENSSL)
```

Comment:
trivial: "USE_OPENSSL" is a little misleading, since it kind of sounds like we're using libssl (and I can imagine a future where that might actually be an option we offer). Would "SEARCH_LIBCRYPTO" be the better name, like they used in s2n's build script? Or is that even more likely to confuse people?

whatever

Reply:
We can fix it with a follow up, either way it requires a coordinated change to aws-c-cal too